conf/routes

```diff
@@ -87,6 +87,7 @@ DELETE /v3/scheduleplans/:guid     @org.sagebionetworks.bridge.play.controllers.
 GET    /v3/studies                 @org.sagebionetworks.bridge.play.controllers.StudyController.getAllStudies(format: String ?= null)
 POST   /v3/studies                 @org.sagebionetworks.bridge.play.controllers.StudyController.createStudy 
 GET    /v3/studies/self            @org.sagebionetworks.bridge.play.controllers.StudyController.getStudyForDeveloper
+GET    /v3/studies/self/public-key @org.sagebionetworks.bridge.play.controllers.StudyController.getStudyPublicKeyAsPem
```

Comment:
Minor: Everywhere else we use camelCase. Should this be publicKey for consistency?